src/server.c

```diff
@@ -3436,7 +3488,7 @@ void call(client *c, int flags) {
         /* Call alsoPropagate() only if at least one of AOF / replication
          * propagation is needed. */
         if (propagate_flags != PROPAGATE_NONE)
-            alsoPropagate(c->db->id,c->argv,c->argc,propagate_flags);
+            alsoPropagateWithPlaceholder(c->db->id,c->argv,c->argc,propagate_flags,index);
     }
 
     /* Restore the old replication flags, since call() can be executed
```

Comment:
trim the last element of the ops array (it's either been replicated or it's a placeholder)

Reply:
add test: module does RM_Call(write-cmd) inside a key-miss notification. the read command that caused key-miss should not be replicated.